showQueueDuration.js

```diff
@@ -9,22 +9,24 @@
 // are added to Spotify. This is to ensure compatibility
 // with older Spotify versions
 const QUEUE_H1_CLASSES = [
-  ".main-type-canon",
-  ".DG9CsoFIptJhAneKoo_F",
-  ".queue-queuePage-header",
+	".main-type-canon",
+	".DG9CsoFIptJhAneKoo_F",
+	".queue-queuePage-header",
+	".mP6tR7IgjiamGqpBW5ai"
```

Comment:
This CSS class wasn't in my Spotify version (MacOS 14.0, spicetify 2.38.4, Spotify 1.2.40.599). In fact, I didn't find any class unique to the "queue" title that would allow me to select only it...

I found that this one did the trick though: The first `h1` in the div with class `vLZJk3f3zoMmc3u9QMrc`:

```
.vLZJk3f3zoMmc3u9QMrc > h1
```

I'll add that one too and rename this to `QUEUE_SELECTORS` to be more general 🙂